indico/modules/events/papers/templates/management/cfp_actions/_base.html

```diff
@@ -0,0 +1,27 @@
+{% extends '_module_actions_base.html' %}
+
+{% block icon_classes %}icon-file{% endblock %}
+
+{% block buttons %}
+    {% set title -%}
+        {%- if cfp.start_dt -%}
+            {%- trans %}Reschedule call for papers{% endtrans -%}
+        {%- else -%}
+            {%- trans %}Schedule call for papers{% endtrans -%}
+        {%- endif -%}
+    {%- endset -%}
+    <div class="group">
```

Comment:
No need for the class `.group` when it only contains 1 element.
https://github.com/indico/indico/wiki/TL-02:-Toolbars#organizing-toolbar-contents